payjoin/src/send/mod.rs

```diff
@@ -235,20 +235,14 @@ impl Configuration {
 /// Represents data that needs to be transmitted to the receiver.
 ///
 /// You need to send this request over HTTP(S) to the receiver.
+/// This is full URL with scheme etc - you can pass it right to `reqwest` or a similar library.
+/// Bytes to be sent to the receiver.
+///
+/// This is properly encoded PSBT, already in base64. You only need to make sure `Content-Type`
+/// is `text/plain` and `Content-Length` is `body.len()` (most libraries do the latter
+/// automatically).
 #[non_exhaustive]
-pub struct Request {
-    /// URL to send the request to.
-    ///
-    /// This is full URL with scheme etc - you can pass it right to `reqwest` or a similar library.
-    pub url: Url,
-
-    /// Bytes to be sent to the receiver.
-    ///
-    /// This is properly encoded PSBT, already in base64. You only need to make sure `Content-Type`
-    /// is `text/plain` and `Content-Length` is `body.len()` (most libraries do the latter
-    /// automatically).
-    pub body: Vec<u8>,
-}
+pub struct Request(pub Url, pub Vec<u8>);
```

Comment:
Another reason to keep the names is to keep the documentation. Adding `pub context: context,` to `Request` with a docstring should make it easier to handle downstream